pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -730,6 +730,22 @@ public void closeFailed(ManagedLedgerException exception, Object ctx) {
         return messageDeduplication.checkStatus();
     }
 
+    private CompletableFuture<Void> checkPersistencePolicies() {
+        DestinationName topicName = DestinationName.get(topic);
+        CompletableFuture<Void> future = new CompletableFuture<>();
+        brokerService.getManagedLedgerConfig(topicName).thenAccept(config -> {
+            // update managed-ledger config and managed-cursor.markDeleteRate
+            this.ledger.setConfig(config);
+            this.ledger.getCursors().forEach(c -> c.setThrottleMarkDelete(config.getThrottleMarkDelete()));
```

Comment:
Why treating the mark-delete throttle rate in a different way compared to other configs? 
Could we just leave the `this.ledger.setConfig(config)` and then leave resetting the rate limiter to the`ManagedLedgerImpl` ?

Reply:
yes, make sense.